messages/list/messages.go

```diff
@@ -0,0 +1,8 @@
+package list
+
+var (
+	Usage            = "list [flags]"
```

Comment:
I dont know if the [flags] makes sense anymore... since list became the primary command

Reply:
even though it doesn't have specific flags for it, it does receive some global flags, do we have anything defined for when we're going to show them? I don't see a problem with showing that it's possible to insert some flags 👍🏽